src/client_backend/client_backend.h

```diff
@@ -28,6 +28,7 @@
 #include <rapidjson/document.h>
 #include <rapidjson/rapidjson.h>
 
+#include <chrono>
```

Comment:
We don't have time for this!

![image](https://github.com/user-attachments/assets/c9c1a271-7af0-49b7-ac79-66b1e17c077a)